src/Sulu/Bundle/AdminBundle/Resources/js/components/Popover/Popover.js

```diff
@@ -54,22 +57,38 @@ class Popover extends React.Component<Props> {
             this.setPopoverSize(0, 0);
             this.updateDimensions();
         });
+
+        if (this.props.open) {
```

Comment:
this implementation corresponds to the implementation in the `Overlay` component: https://github.com/sulu/sulu/blob/a00ad7ad1a1c5cffe245c586dbcd360e4beabff8/src/Sulu/Bundle/AdminBundle/Resources/js/components/Overlay/Overlay.js#L55-L57